packages/reports/addon/templates/components/navi-request-column-config/metric.hbs

```diff
@@ -0,0 +1,38 @@
+{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="navi-request-column-config" ...attributes>
+  <div class="navi-request-column-config__header">
+    <span class="navi-request-column-config__header-label">Column Configuration</span>
+    <span class="navi-request-column-config__header-icons">
+      <NaviIcon @icon="star-o" class="navi-request-column-config__star-icon" role="button"/>
+      <NaviIcon @icon="times" class="navi-request-column-config__exit-icon" role="button" {{action this.onClose}}/>
+    </span>
+  </div>
+  <div class="navi-request-column-config__body">
+    <div class="navi-request-column-config__option navi-request-column-config__column-name">
+      <label for="columnName" class="navi-request-column-config__column-name-label navi-request-column-config__option-label">Column Name</label>
+      <Input 
+        class="navi-request-column-config__column-name-input navi-request-column-config__option-input"
+        @id="columnName" 
+        @value={{@column.displayName}}
+        @key-up={{action @onUpdateColumnName}}
+      />
+    </div>
```

Comment:
can we keep the above in base or some wrapping component and use for dimension/date-time?

Reply:
Done 👍